src/handlefileupload.js

```diff
@@ -22,24 +22,34 @@ var initSocket = require('./socket').initSocket;
 exports.handleFileUpload = function(type, token, model, file, contentType, callback, onend) {
   // Set currentlyDisplaying to prevent other sockets from opening
   localStorage.setItem('currentlyDisplaying', type);
-
+  
   $.subscribe('progress', function(evt, data) {
     console.log('progress: ', data);
   });
 
   console.log('contentType', contentType);
 
-  var baseString = '';
+  var result = {};
```

Comment:
Not a blocker, but I think this should be renamed. I was getting confused later in the code thinking that `result` was the JSON object received from the service. Maybe `state` or `output`?